canonical-web-and-design/ubuntu.com
# Reorder the microcerts table

## Done

- Re-order the mircrocerts table to match the comment in the copy doc

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cube/microcerts
- Check the list of microcerts is in the correct order from the [copy doc](https://docs.google.com/document/d/1IHmr2SIxcEKVNN2sA-z_32PV0fCXi38pmeVP7k7O1mM/edit?ts=60463539#heading=h.2f6a32osnzfh) comment


## Issue / Card

Fixes #9367
